index.html

```diff
@@ -2022,73 +2022,62 @@
 
-A <dfn data-lt="vulnerable">vulnerable person</dfn> is a [=person=] who may be unable to
-exercise sufficient self-determination in a [=context=]. Amongst other things, they should
+A <dfn data-lt="vulnerable">vulnerable person</dfn> in a particular [=context=]
+is a [=person=] whose ability to make their own choices can be taken away more
+easily than usual. Among other things, they should
 be treated with greater default privacy protections and may be considered unable to
 [=consent=] to various interactions with a system.
 People can be vulnerable for different reasons, for example because they are children,
 are employees with respect to their employers, are facing a steep asymmetry of power,
 are people in some situations of intellectual or psychological impairment, are
-refugees, etc.
+refugees, etc. See [[[#vulnerability]]].
 
 ## Contexts {#context}
 
 A <dfn>context</dfn> is a physical or digital environment in which [=people=] interact with other
 [=actors=], and which the [=people=] understand as distinct from other [=contexts=].
 
 A [=context=] is not defined in terms of who owns or controls it. Sharing
-[=data=] between different [=contexts=] of a single company is
+[=data=] between different [=contexts=] of a single company can be
 a [=privacy violation=], just as if the same data were shared between unrelated [=actors=].
 
 ## Server-Side Actors {#parties}
 
 An <dfn>actor</dfn> is an entity that a [=person=] can reasonably understand as a single "thing"
 they're interacting with. [=Actors=] can be [=people=] or collective entities like companies,
-associations, or governmental bodies. Uses of this document in a particular domain are expected to
-describe how the core concepts of that domain combine into a [=user=]-comprehensible [=actor=], and
-those refined definitions are likely to differ between domains.
+associations, or governmental bodies.
 
 [=User agents=] tend to explain to [=people=] which [=origin=] or [=site=] provided the
-web page they're looking at. The [=actor=] that controls this [=origin=] or [=site=] is
+web page they're looking at. The [=actor=] that makes or delegates decisions
+about the content and [=data processing=] on this [=origin=] or [=site=] is
 known as the web page's <dfn data-dfn-for="page">first party</dfn>. When a [=person=]
-interacts with a UI element on a web page, the <dfn>first party</dfn> of that interaction
-is usually the web page's [=page/first party=]. However, if a different [=actor=] controls
-how data collected with
-the UI element is used, and a reasonable person with a realistic cognitive budget would realize
+interacts with a part of a web page, the <dfn>first party</dfn> of that interaction
+is usually the web page's [=page/first party=].
+However, if a different [=actor=] makes the decisions about how that part of the
+page works, and a reasonable person with a realistic cognitive budget would realize
```

Comment:
is there a simpler / more self-explanatory phrase than "cognitive budget"?

Reply:
Hmmm, maybe:

```suggestion
page works, and a reasonable person with a realistic amount of time and energy to think about the situation would realize
```

Although I'm a little worried that the sentence is getting long enough that "would realize" is disconnected from its subject.

Reply:
I thought about flipping the order of the clauses, but that doesn't make it more readable either. You could omit "to think about the situation." 
```suggestion
page works, and a reasonable person with a realistic amount of time and energy would realize
```

Reply:
+1 to @wseltzer's version